tests/acceptance/bootstrap/SearchContext.php

```diff
@@ -221,4 +246,31 @@ public function theSearchResultShouldContainEntriesWithHighlight(
 			);
 		}
 	}
+
+	/**
+	 * @When /^user "([^"]*)" searches for "([^"]*)" inside (folder|space) "([^"]*)" using the WebDAV API$/
+	 * @When /^user "([^"]*)" searches for "([^"]*)" inside (folder) "([^"]*)" in space "([^"]*)" using the WebDAV API$/
+	 *
+	 * @param string $user
+	 * @param string $pattern
+	 * @param string $scopeType
+	 * @param string|null $scope
+	 * @param string|null $spaceName
+	 *
+	 * @return void
+	 * @throws Exception|GuzzleException
+	 */
+	public function userSearchesInsideFolderOrSpaceUsingWebDavAPI(
+		string  $user,
+		string  $pattern,
+		string  $scopeType,
+		?string $scope = null,
+		?string $spaceName = null,
```

Comment:
`scope` is a required arg
```suggestion
	 * @param string $scope
	 * @param string|null $spaceName
	 *
	 * @return void
	 * @throws Exception|GuzzleException
	 */
	public function userSearchesInsideFolderOrSpaceUsingWebDavAPI(
		string  $user,
		string  $pattern,
		string  $scopeType,
		string $scope,
		?string $spaceName = null,
```